elasticsearch/native/src/main/java/org/apache/metamodel/elasticsearch/nativeclient/ElasticSearchDataContextFactory.java

```diff
@@ -131,10 +135,16 @@ public DataContext create(DataContextProperties properties, ResourceFactoryRegis
     }
 
     private Client createTransportClient(DataContextProperties properties) {
-        final Settings settings = Settings.builder().put().put("name", "MetaModel").put("cluster.name", getCluster(properties)).build();
+        final Settings settings = Settings
+                .builder()
+                .put("name", "MetaModel")
```

Comment:
I had to remove one "put()" here to get it to work, otherwise I wouldn't have touched this piece of code at all. Since the nativeclient will be deprected as part of this PR, I'd rather change as little of the internal implementation as possible.